### ISSUE TITLE
Avoiding the "400 POST parameters exceeds the maximum limit." CF error

### ISSUE THREAD

**Issue by damien-git** [blueriver/MuraCMS#2629]
This is not strictly speaking a bug, but we ran into an issue that could potentially be avoided.

We have a large number of groups in the same site, and Mura uses a large form with as many inputs as there are groups in the site to change content permissions.  The number of parameters in the POST request goes beyond ColdFusion default, 100. We were able to change that CF setting to avoid this error, but some users might not be able to change it if they don't control CF parameters. Also, there is always the risk to run into the issue after adding a new goup.

Ideally, The number of POST parameters used by Mura should never go beyond that default maximum of 100.

**Comment by gowestweb** [comment]
The fix is to change the CF setting.

**Comment by mattlevine** [comment]
Do you have a lot of extended attributes or custom UI adding to what is there by default?

**Comment by damien-git** [comment]
no, just many groups (we are putting all users and groups in the same site and sharing it for users and groups for all sites, so that we don't have duplicate users in different sites; this is why we end up with many groups in the same site)

**Comment by ASKemp** [comment]
As noted, this is a documented CF setting.
